notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>() no need, same for all tables</p><p> </p><p>what do we mean by artifacts here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>spurious signal that is supposedly not of neural origin. </p>

Reply:
<p>I think that's a good definition, maybe include that in the text?</p><p> </p>